client/components/pagination/index.js

```diff
@@ -26,27 +26,14 @@ function keyListener( methodToCall, event ) {
 class Pagination extends Component {
 	constructor( props ) {
 		super( props );
-		this.state = {
-			pageInputValue: props.page,
-		};
```

Comment:
Good refactor here. This is an anti-pattern, https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#anti-pattern-unconditionally-copying-props-to-state